src/features/query-bar/QueryBar.js

```diff
@@ -27,12 +27,17 @@ const QueryBar = ({
     onFilterChange,
     onTemplateChange,
     templates,
-}: Props) => (
-    <section className="metadata-view-query-bar">
-        <TemplateButton activeTemplate={activeTemplate} onTemplateChange={onTemplateChange} templates={templates} />
-        <FilterButton columns={columns} conditions={conditions} onFilterChange={onFilterChange} />
-        <ColumnButton columns={columns} onColumnChange={onColumnChange} template={activeTemplate} />
-    </section>
-);
+}: Props) => {
+    const columnsWithoutItems = columns && columns.filter(column => column.source !== 'item');
```

Comment:
I wonder if we should name what it is instead of what it omits e.g. "only the metadata columns", "metadata columns".